### ISSUE TITLE
koppelen geslotenverklaring met VoertuiglengteGroterDan, foutmelding unit

### ISSUE THREAD

**Issue by ElisabethDeVries** [Stichting-CROW/verkeersborden#351]
ik kan geen unit toevoegen, hij blijft een foutmelding geven?

**Comment by gulaycanbaloglu** [comment]
I just added the link with unit for aspect, does it look fine?

**Comment by ElisabethDeVries** [comment]
@gulaycanbaloglu klopt het dat ik voor elk aspect met een unit eerst jouw hulp moet vragen? Want er staan er nog meer die ik niet gekoppeld krijg

Geslotenverklaring > Voertuiglengte, -breedte, -hoogte, aslast (ton) en aslastsomgroterdan (ton)

**Comment by gulaycanbaloglu** [comment]
nee, it should be possible for you to do that yourself as well. You should create the unit in the units tab first, then you can add it while linking the aspect to a concept.

**Comment by ElisabethDeVries** [comment]
linksonderin het unit veld
